api_tests/institutions/views/test_institution_summary_metrics.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+import datetime
+
+from api.base.settings.defaults import API_BASE
+from osf_tests.factories import (
+    AuthUserFactory,
+    InstitutionFactory
+)
+from osf.metrics import InstitutionProjectCounts
+
+
+@pytest.mark.es
+@pytest.mark.django_db
+class TestInstitutionSummaryMetrics:
+
+    @pytest.fixture()
+    def institution(self):
+        return InstitutionFactory()
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def admin(self, institution):
+        user = AuthUserFactory()
+        group = institution.get_group('institutional_admins')
```

Comment:
Not part of this PR, but I think there's supposed to be an institutional admin group for each institution. Otherwise anyone who is an admin could view any institution's summary information, which is not appropriate.

Reply:
`institution.get_group('institutional_admins')` returns a group in the format `institution_<institution._id>_institutional_admins` in my local shell so the group is specific to the institution which indicates to me that it should be fine - does that sound correct to you?

Reply:
Yes, very good. That makes much more sense than what I was thinking first thing this morning. :)